### ISSUE TITLE
Crashed while exploring with volerglider

### ISSUE THREAD

**Issue by Tiskeris** [DarkPacks/SevTech-Ages#4256]
## Bug Description

I was playing sevtech ages of the sky. Exploring with my volarglider and suddenly the game crashed. I tried getting into the world and loading the game but it just loads and insta crashes again.

### Did this behavior use to work in the previous version?

This is the only version i was playing : 3.2.0-alpha.2-skyblock

## Steps to Reproduce (for bugs)

1.
2.
3.

## Logs

* Client/Server Log:https://paste.atlauncher.com/view/8e50f1e9
* Crash Log:https://pastebin.com/chLX5iQ8

## Client Information
* Modpack Version:3.2.0-alpha.2-skyblock
* Java Version:261
* Launcher Used:AT
* Memory Allocated:6GB
* Server/LAN/Single Player:Single
* Optifine Installed:no
* Shaders Enabled:no

## World Information
* Modpack Version world created in:3.2.0-alpha.2-skyblock
* Additional Content Installed:

**Comment by InariusTheRisen** [comment]
I'm also having this issue; have tried setting java config to remove erroring entities and tile entities but no luck so far.

**Comment by Om3ga12** [comment]
Yes I am having the same issue. My friend and I were playing on LAN together, his game crashed while I was using the Voraglider. Every time I try to reconnect his game freezes and crashes, and I dont even get the chance to join.

**Comment by sam-kirby** [comment]
Adding report from Discord, includes seed - may help with reproduction

> about 30 min after i hit age 1, i went out flying with the starter kite, and out of nowhere i crashed. again and again ive tried to re-join my world, but crash keeps happening.

> Modpack Version: SevTech-Ages-of-the-Sky_3.2.0-alpha.2-skyblock.zip
> Java Version: Version 8 Update 261 (build 1.8.0_261-b12) 64-bit
> Launcher Used: Twitch, Import Profile
> Memory Allocated: 6400 MB RAM
> Server/LAN/Single Player: Single Player
> Optifine Installed: No
> Shaders Enabled: No

> https://pastebin.com/5qLDaYD8 Initial Crash Report.
> https://pastebin.com/Wsb2Cwpf Last 700 Lines of my Log. (just in case)
> https://pastebin.com/tu0B6tMp 1st Crash After Initial Crash
> https://pastebin.com/gWjpwZUN 2nd Crash After Initial Crash

> Note: i know have a better idea of what caused the above, after getting to age 1, when i go to a certain XYZ my client crashes.
deeper explanation, at a place i already explored and loaded chunks, when i get in range of "still unknown" my game crash, this "spot" is right next to a birch biome not far from spawn at all, and the fact that if i ever go "that" way in my world my game crash, makes progression harder. as sailors would say if they went past the know mapped sea; Here There Be Dragons" but in my case, HARD VIOLENT crashes LUL my seed is: -8709018635197482015 if anyone else can recreate this, i dont have exact cords of where this happends, but its not hard to fly around and find the point as its so close and a birch biome. NOTE: this did not happen in stage 0, only after stage 1. but i do know its on the right side of the jungle to the east of the spawn thx to sunflowers

**Comment by ninja5quirrelz** [comment]
Any fixes for this? I've tried loading a backup but the game still crashes.

**Comment by sam-kirby** [comment]
Wait for the next update, hopefully a fix will be ready by then.

**Comment by Darkosto** [comment]
Fix should be in Alpha 3. Waiting on another fix before we post it.

**Comment by MikeHekman** [comment]
I have the same problem, only it happened to me in age 2.

**Comment by spazcore123** [comment]
Yeah i have this same issue, I got 2 concerns, is my world done for, and is there a fix?

**Comment by sam-kirby** [comment]
https://github.com/DarkPacks/SevTech-Ages/issues/4256#issuecomment-696300205

Yes, there is a fix, it will be released very soon. Development on alpha 3 is wrapping up on Darkosto's stream now.

If you want a more immediate fix, restore a backup.

**Comment by Om3ga12** [comment]
I just downloaded the newest version with my friend and we still crash once I join the game.

**Comment by sam-kirby** [comment]
@Om3ga12 Please create a new issue with all requested details. You are likely having a different issue.

This issue should be resolved as of alpha 4. If anyone is still experiencing it in that version, please open a new issue.
